source/releasenotes/2024-07-01-workflow-logs.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Introducing Workflow Logs: General availability "
+published_date: "2024-07-01"
+categories: [new-feature, user-interface]
+---
+
+We’re excited to announce the launch of Workflow Logs, a powerful new feature enhancing visibility and streamlining troubleshooting operations on the Pantheon platform. Accessible directly from your site dashboard, next to the Support tab located in the top right corner, Workflow Logs provide critical insights for effectively managing your workflows.
+
+## Key features:
+* **Status overview**: View key information on the status of workflows, including progress statuses, along with additional details such as start time, end time, initiating team member, and environment.
+* **In-depth details**: Access code sync, error details, build logs, and Quicksilver posted messages from the expandable sections.
+* **Filtering**: The workflows displayed comprise a curated list of user-relevant workflows. Further narrow your focus by utilizing filters to quickly find specific workflows by environment or completion status.
+* **Archiving**: Export all visible workflows to a CSV file for archival purposes with the 'Download to CSV' button.
+
+<!-- TODO: Add image from gdoc draft -->
+
+Our team will continuously enhance Workflow Logs through a planned list of improvements and ongoing feedback collection.
```

Comment:
In the EA release note we specified that feedback would be collected via the EA program. How will feedback be collected in GA?

Reply:
What's the standard treatment for for updates on features in general access? Do we always include a note about feedback?

Reply:
@moreCoffie My preference is not to mention feedback collection unless we're actively seeking and collecting feedback from a given feature with a defined process (e.g., have your customer fill out this form, or email that person).

Reply:
Yes, I prefer that too.